suites/apis/getCoremetadataTest.js

```diff
@@ -1,46 +1,59 @@
 Feature('GetCoreMetadata');
 
-// // test data
-// let valid_file;
-// let invalid_id_file;
-//
-// Scenario('test core metadata', async(I) => {
-//   let metadata = await I.getCoremetadata(valid_file, 'json');
-//   I.seeJsonCoremetadata(valid_file, metadata);
-//
-//   metadata = await I.getCoremetadata(valid_file, 'bibtex');
-//   I.seeBibtexCoremetadata(valid_file, metadata);
-// });
-//
-// Scenario('test core metadata invalid object_id', async(I) => {
-//   let data = await I.getCoremetadata(invalid_id_file, 'json');
-//   I.seePidginError(data);
-// });
-//
-// Scenario('test core metadata no permission', async(I) => {
-//   let invalid_token = { 'Authorization': 'invalid' };
-//   let data = await I.getCoremetadata(valid_file, 'json', invalid_token);
-//   I.seePidginError(data);
-// });
-//
-// BeforeSuite(async (I) => {
-//   // try to clean up any leftover nodes
-//   await I.findDeleteAllNodes();
-//
-//   // submit test data
-//   let nodes = I.getNodePathToFile();
-//   await I.addNodes(I.getSheepdogRoot(), Object.values(nodes));
-//   I.seeAllNodesAddSuccess(Object.values(nodes));
-//
-//   valid_file = {
-//     "data": util.extractFile(nodes)
-//   };
-//   await I.submitFile(I.getSheepdogRoot(), valid_file);
-//
-//   invalid_id_file = util.clone(valid_file);
-//   invalid_id_file.did = 'invalid_object_id';
-// });
-//
-// AfterSuite(async (I) => {
-//   await I.findDeleteAllNodes();
-// });
+// test data
+let valid_file;
+let invalid_id_file;
+
+Scenario('test core metadata', async(pidgin, users) => {
+  let metadata = await pidgin.do.getCoremetadata(valid_file, 'application/json', users.mainAcct.accessTokenHeader);
+  pidgin.ask.seeJsonCoremetadata(valid_file, metadata);
+
+  metadata = await pidgin.do.getCoremetadata(valid_file, 'x-bibtex', users.mainAcct.accessTokenHeader);
+  pidgin.ask.seeBibtexCoremetadata(valid_file, metadata);
+
+  metadata = await pidgin.do.getCoremetadata(valid_file, 'application/vnd.schemaorg.ld+json', users.mainAcct.accessTokenHeader);
+  pidgin.ask.seeSchemaorgdata(valid_file, metadata);
```

Comment:
since this will be implemented later, let's remove this block and the `seeSchemaorgdata() function`